test/JDBC/expected/Test_sp_renamedb-vu-verify.out

```diff
@@ -0,0 +1,359 @@
+
+
+-- tsql user=sp_renamedb_login1 password=1234
+-- now disconnect from that database in the 2nd connection and test alter again
+select db_name()
+go
+~~START~~
+nvarchar
+sp_renamedb_database2
+~~END~~
+
+use master
+go
+
+-- tsql
+exec sp_renamedb 'sp_renamedb_database2', 'sp_renamedb_database1';
+go
+
+drop function sp_renamedb_function
+drop procedure sp_renamedb_procedure
+go
+
+-- testing invalid args for sp_renamedb
+create database sp_renamedb_db1_temp;
+go
+-- NULL old_db_name
+exec sp_renamedb NULL, 'sp_renamedb_db1_temp';
+go
+~~ERROR (Code: 33557097)~~
+
+~~ERROR (Message: The database '(null)' does not exist. Supply a valid database name. To see available databases, use sys.databases.)~~
+
+-- NULL new_db_name
+exec sp_renamedb 'sp_renamedb_db1_temp  ', NULL;
+go
+~~ERROR (Code: 33557097)~~
+
+~~ERROR (Message: The value for the @newname parameter contains invalid characters or violates a basic restriction ((null)).)~~
+
+-- both NULL
+exec sp_renamedb NULL, NULL;
+go
+~~ERROR (Code: 33557097)~~
+
+~~ERROR (Message: The database '(null)' does not exist. Supply a valid database name. To see available databases, use sys.databases.)~~
+
+-- empty string old_db_name
+exec sp_renamedb '', '     sp_renamedb_db1_temp';
+go
+~~ERROR (Code: 33557097)~~
+
+~~ERROR (Message: The value for the @objname parameter contains invalid characters or violates a basic restriction (()).)~~
+
+-- empty string new_db_name
+exec sp_renamedb '    sp_renamedb_db1_temp', '';
+go
+~~ERROR (Code: 33557097)~~
+
+~~ERROR (Message: The value for the @newname parameter contains invalid characters or violates a basic restriction (()).)~~
+
+-- both empty string
+exec sp_renamedb '    ', ' '
+go
+~~ERROR (Code: 33557097)~~
+
+~~ERROR (Message: The value for the @objname parameter contains invalid characters or violates a basic restriction (()).)~~
+
+-- 3 part names
+exec sp_renamedb ' sp_renamedb_db1_temp.sch1.t1', 'db2      '
+go
+~~ERROR (Code: 33557097)~~
+
+~~ERROR (Message: The value for the @objname parameter contains invalid characters or violates a basic restriction (( sp_renamedb_db1_temp.sch1.t1)).)~~
+
+-- 3 part names
+exec sp_renamedb 'sp_renamedb_db1_temp', 'db2.sch2.t2'
+go
+~~ERROR (Code: 33557097)~~
+
+~~ERROR (Message: The value for the @newname parameter contains invalid characters or violates a basic restriction ((db2.sch2.t2)).)~~
+
+-- 3 part names
+exec sp_renamedb 'db1.sch1.t1', 'db2.sch2.t2'
+go
+~~ERROR (Code: 33557097)~~
+
+~~ERROR (Message: The value for the @objname parameter contains invalid characters or violates a basic restriction ((db1.sch1.t1)).)~~
+
+-- valid input, should get db does not exist
+exec sp_renamedb '[db1.sch1.t1]', 'sp_renamedb_db1_temp      '
+go
+~~ERROR (Code: 33557097)~~
+
+~~ERROR (Message: Database db1.sch1.t1 does not exist. Make sure that the name is entered correctly.)~~
+
+-- valid input, should get db does not exist
```

Comment:
Please add cases with invalid chars as well.

Reply:
Spoke offline, we dont need to add more tests for this